system/Pager/Pager.php

```diff
@@ -481,7 +481,7 @@ public function getDetails(string $group = 'default'): array
      *
      * @return Pager
      */
-    public function only(array $queries):Pager
+    public function only(array $queries):self
```

Comment:
Probably covered in some other rule but:
```suggestion
    public function only(array $queries): self
```

Reply:
This is covered by return type declaration fixer.